examples/standard_platformer/lib/coin.dart

```diff
@@ -24,10 +25,11 @@ class Coin extends PhysicalEntity {
     );
   }
 
+  final TiledObject tiledObject;
+
   @override
   void onRemove() {
     super.onRemove();
-
     FlameAudio.play('coin.wav');
   }
 
```

Comment:
For some reason the `stepTime` on the coin animation needs to be changed from `2.5` to `0.2` or something around there. Not sure why that changed between versions